### PR TITLE
updated the readme

### PR DESCRIPTION
**Pull request by alifarhid**

### CODE REVIEW

File: 07-Docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Copy the two Python files to the container
+COPY pi.ipynb quickstart_notebook.ipynb /app/
+
+# Expose port 8888 for the Jupyter Notebook
+EXPOSE 8888
+
+ENV PATH=/opt/conda/bin:$PATH
+
+# Start IPyParallel cluster with max 8 cores
+CMD ["sh", "-c", "ipcluster start --profile=default --n=$(($(nproc)-1)) --nthreads=$(($(nproc)-1)) --engines=$(($(nproc)>8 ? 8 : $(nproc)-1)) & jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root"]
```
Comment (hadia206) [inline 07-Docker/Dockerfile:37]: newline

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 .ipynb_checkpoints
 *.pq
-
-
+*.parquet
+.idea
```
Comment (hadia206) [inline .gitignore:4]: Same. newline

File: 07-Docker/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- You also need to run the Docker daemon (e.g., run the Docker Desktop).
+- Log in if you want to push a new image to dockerhub. 
+
+## Start the Bodo Community Edition
+
+The bodo docker image is available at https://hub.docker.com/r/bodoaidocker/bodo-community-edition
+
+You can follow the instruction there, or just continue reading here. 
+
+All you need to do is to run the following command:
```
Comment (hadia206) [inline 07-Docker/README.md:18]: Shouldn't they do `docker pull` first?

**Review by hadia206 — COMMENTED** [review]
I have a question. Otherwise looks good to me.

File: 07-Docker/README.md (hunk 3 above)
Comment (alifarhid) [reply 07-Docker/README.md:18]: added

File: 07-Docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
+FROM condaforge/mambaforge as build
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV JUPYTER_ENABLE_LAB=yes
+
+RUN mamba install bodosql=2023.12.1 -c bodo.ai/label/dev -c conda-forge
+RUN mamba install -y ipyparallel=8.1 jupyterlab=3 -c conda-forge
+
+#RUN conda remove --force --yes mpich mpi
```
Comment (hadia206) [inline 07-Docker/Dockerfile:9]: Can we remove this commented code?

**Review by hadia206 — APPROVED** [review]
Looks good to me. Thanks Ali.

**Pull request merged**